mifosng-android/src/main/res/layout/row_client_list_item.xml

```diff
@@ -28,7 +28,6 @@
             android:id="@+id/tv_product_short_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/tv_clientName"
             android:text=""
```

Comment:
Isn't this obsolete too?

Reply:
Yeah, it is. Either we could delete it or give some text like other Text Views

Reply:
I included the text for that text view.